{
    "metadata": {
        "@context": "https://openvex.dev/ns/v0.2.0",
        "@id": "https://github.com/canonical/ubuntu-security-notices/blob/main/vex/usn/USN-6888-2",
        "author": "Canonical Ltd.",
        "timestamp": "2025-01-13T11:52:43.503719",
        "version": 2
    },
    "statements": [
        {
            "vulnerability": {
                "@id": "https://ubuntu.com/security/notices/USN-6888-2",
                "name": "USN-6888-2",
                "description": "USN-6888-1 fixed several vulnerabilities in Django. This update provides the corresponding update for Ubuntu 18.04 LTS.  Original advisory details:   Elias Myllym\u00e4ki discovered that Django incorrectly handled certain inputs  with a large number of brackets. A remote attacker could possibly use this  issue to cause Django to consume resources or stop responding, resulting in  a denial of service. (CVE-2024-38875)   It was discovered that Django incorrectly handled authenticating users with  unusable passwords. A remote attacker could possibly use this issue to  perform a timing attack and enumerate users. (CVE-2024-39329)   Josh Schneier discovered that Django incorrectly handled file path  validation when the storage class is being derived. A remote attacker could  possibly use this issue to save files into arbitrary directories.  (CVE-2024-39330)   It was discovered that Django incorrectly handled certain long strings that  included a specific set of characters. A remote attacker could possibly use  this issue to cause Django to consume resources or stop responding,  resulting in a denial of service. (CVE-2024-39614)",
                "aliases": [
                    "CVE-2024-38875",
                    "CVE-2024-39329",
                    "CVE-2024-39330",
                    "CVE-2024-39614"
                ]
            },
            "timestamp": "2024-07-11 10:41:49.602341",
            "products": [
                {
                    "@id": "pkg:deb/ubuntu/python-django@1:1.11.11-1ubuntu1.21+esm5?arch=source&distro=esm-infra/bionic"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django@1:1.11.11-1ubuntu1.21+esm5?arch=amd64"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django@1:1.11.11-1ubuntu1.21+esm5?arch=arm64"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django@1:1.11.11-1ubuntu1.21+esm5?arch=armhf"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django@1:1.11.11-1ubuntu1.21+esm5?arch=i386"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django@1:1.11.11-1ubuntu1.21+esm5?arch=ppc64el"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django@1:1.11.11-1ubuntu1.21+esm5?arch=s390x"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django-common@1:1.11.11-1ubuntu1.21+esm5?arch=amd64"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django-common@1:1.11.11-1ubuntu1.21+esm5?arch=arm64"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django-common@1:1.11.11-1ubuntu1.21+esm5?arch=armhf"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django-common@1:1.11.11-1ubuntu1.21+esm5?arch=i386"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django-common@1:1.11.11-1ubuntu1.21+esm5?arch=ppc64el"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django-common@1:1.11.11-1ubuntu1.21+esm5?arch=s390x"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django-doc@1:1.11.11-1ubuntu1.21+esm5?arch=amd64"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django-doc@1:1.11.11-1ubuntu1.21+esm5?arch=arm64"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django-doc@1:1.11.11-1ubuntu1.21+esm5?arch=armhf"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django-doc@1:1.11.11-1ubuntu1.21+esm5?arch=i386"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django-doc@1:1.11.11-1ubuntu1.21+esm5?arch=ppc64el"
                },
                {
                    "@id": "pkg:deb/ubuntu/python-django-doc@1:1.11.11-1ubuntu1.21+esm5?arch=s390x"
                },
                {
                    "@id": "pkg:deb/ubuntu/python3-django@1:1.11.11-1ubuntu1.21+esm5?arch=amd64"
                },
                {
                    "@id": "pkg:deb/ubuntu/python3-django@1:1.11.11-1ubuntu1.21+esm5?arch=arm64"
                },
                {
                    "@id": "pkg:deb/ubuntu/python3-django@1:1.11.11-1ubuntu1.21+esm5?arch=armhf"
                },
                {
                    "@id": "pkg:deb/ubuntu/python3-django@1:1.11.11-1ubuntu1.21+esm5?arch=i386"
                },
                {
                    "@id": "pkg:deb/ubuntu/python3-django@1:1.11.11-1ubuntu1.21+esm5?arch=ppc64el"
                },
                {
                    "@id": "pkg:deb/ubuntu/python3-django@1:1.11.11-1ubuntu1.21+esm5?arch=s390x"
                }
            ],
            "status": "fixed",
            "status_notes": "Several security issues were fixed in Django. Available with Ubuntu Pro."
        }
    ]
}